client/grpc/client.go

```diff
@@ -38,7 +38,7 @@ func New(address, certPath string, insecure bool) (client.Client, error) {
 	} else if insecure {
 		opts = append(opts, grpc.WithInsecure())
 	} else {
-		opts = append(opts, grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{})))
+		opts = append(opts, grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{MinVersion: tls.VersionTLS12})))
```

Comment:
@willscott Just flagging that we're putting a minimum now on our TLS connection - I think we should be fine but we should make sure all our partners have TLS1.2 at the very least.

Reply:
We may need to add this to the README. This was something linter required

Reply:
> I'm looking at the status checks on this PR to switch over the required ones for merges to master, and I don't see the new ones as being run. Is this because it's coming from a fork, or is it unexpected we wouldn't see the [`build/test`](https://github.com/drand/drand/pull/813/files#diff-95557d53bf91069e59d82b9d8fcfaf52ec84a762858983edd5ef3c9b3e4c8191R1) action running in this PR?

I added PRs to the events that run that job (build/tests). It should run now.